src/extension.ts

```diff
@@ -96,22 +96,22 @@ export function activate(context: vscode.ExtensionContext) {
 
   createStatusBarItem(context);
 
-  let disposableToggleCodelens =
-      vscode.commands.registerCommand('onevscode.toggle-codelens', () => {
-        let codelensState =
-            vscode.workspace.getConfiguration('one-vscode').get('enableCodelens', true);
-        vscode.workspace.getConfiguration('one-vscode')
-            .update('enableCodelens', !codelensState, true);
-      });
-  context.subscriptions.push(disposableToggleCodelens);
-
-  let codelens = new CodelensProvider(context);
-  let disposableCodelens = vscode.languages.registerCodeLensProvider('ini', codelens);
-  context.subscriptions.push(disposableCodelens);
-
-  let hover = new HoverProvider();
-  let disposableHover = vscode.languages.registerHoverProvider('ini', hover);
-  context.subscriptions.push(disposableHover);
+  // let disposableToggleCodelens =
```

Comment:
Is it what we talked about `deprecation`?

Reply:
Yes, but i think this is out of scope for this draft :)
Please ignore these removals!